core/src/odbc_uri.rs

```diff
@@ -22,6 +22,7 @@ pub const UID: &str = "uid";
 pub const URI: &str = "uri";
 pub const APPNAME: &str = "appname";
 pub const LOGLEVEL: &str = "loglevel";
+pub const SIMPLE: &str = "simple_types_only";
```

Comment:
```suggestion
pub const SIMPLE_TYPES_ONLY: &str = "simple_types_only";
```
[nit] everywhere else the name we use for the constant is the same as the constant, so I'm suggesting this just for consistency.